### PR TITLE
Ensure user data exists for `isFirstAppLaunch` method

### PR DESCRIPTION
**Pull request by joecohens**
While updating https://github.com/joecohens/electron-first-run thanks to your tweet, I've found the first test failed a always. Digging a little further I've found that `userData` can be missing, more info here:  https://github.com/Microsoft/vscode/issues/19662 

Thanks and I hope it helps :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -148,15 +149,24 @@ exports.disableZoom = (win = activeWindow()) => {
 exports.appLaunchTimestamp = Date.now();
 
 if (is.main) {
-	exports.isFirstAppLaunch = () => {
+	const isFirstAppLaunch = () => {
 		const fs = require('fs');
-		const checkFile = path.join(api.app.getPath('userData'), '.electron-util--has-app-launched');
+		const checkFile = path.join(api.app.getPath('userData'), 'FirstRun', '.electron-util--has-app-launched');
```
Comment (sindresorhus) [inline index.js]: Don't do unrelated changes.

File: index.js [hunk 2]
```diff
@@ -148,15 +149,24 @@ exports.disableZoom = (win = activeWindow()) => {
...
 		if (fs.existsSync(checkFile)) {
 			return false;
 		}
 
-		fs.writeFileSync(checkFile, '');
+		try {
+			fs.writeFileSync(checkFile, '');
+		} catch (error) {
+			if (error.code === 'ENOENT') {
+				makeDir.sync(path.join(api.app.getPath('userData'), 'FirstRun'));
```
Comment (sindresorhus) [inline index.js]: Since we only need to create a directory for `api.app.getPath('userData')`, which is just one directory, you can just use `fs.mkdir`.

Comment (joecohens) [reply index.js]: Thanks for the comments, I've made the updates :)
